dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -226,6 +227,16 @@
  *         </td>
  *     </tr>
  *     <tr>
+ *         <td>{@code uriCompliance}</td>
+ *         <td>RFC7230</td>
+ *         <td>
+ *             This sets the uri compliance level used by Jetty when parsing http, this can be useful when
+ *             attempting to avoid breaking changes with Jetty 10 and onward;
+ *
+ *             Possible values are set forth in the org.eclipse.jetty.http.UriCompliance enum.
+ *         </td>
+ *     </tr> *
```

Comment:
The additional star at the line end should be removed to have a cleaner javadoc.